examples/tutorials/05_ordinary_torch_models.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<p>Why is it split into 2 cells?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>